sct.py

```diff
@@ -1594,6 +1596,25 @@ def generate_parallel_timelines_report(logdir: str | None, test_id: str | None)
     pt_report_generator.generate_full_report()
 
 
+@cli.command("create-es-index", help="Create ElasticSearch index with mapping ")
+@click.option("-n", "--name", envvar='SCT_ES_INDEX_NAME', help="ES index name")
+@click.option("-dt", "--doc-type", envvar='SCT_ES_DOC_TYPE', default="")
+@click.option("-f", "--mapping-file", envvar='SCT_MAPPING_FILEPATH', type=click.Path(exists=True),
+              help="Full path to es index mapping file")
+def create_es_index(name: str, doc_type: str, mapping_file: str) -> None:
+    add_file_logger()
+
+    mapping_data = get_mapping(mapping_file)
+    if mapping_data is None:
+        click.secho(f"File {mapping_file} with mapping  not found")
+        sys.exit(1)
+    status = create_index(index_name=name, doc_type=doc_type, mappings=mapping_data)
+    if not status:
+        click.secho(f"Index {name} was not created")
```

Comment:
I'm not sure reason of the failure would be shown like this, just raise the exception, and whom is using it would need to handle it.

Reply:
removed